src/components/DogList.jsx

```diff
@@ -0,0 +1,15 @@
+import Dog from './Dog';
+import './DogList.css';
+
+const DogList = () => {
+  return  (
+    <div>
+      <h2 className='dogListHeader'> Dog List </h2>
+      <Dog />
+      <Dog />
+      <Dog />
+    </div>
+)
```

Comment:
How about putting the class on the outer div and then using more specific style rules to target the heading?
```suggestion
  return (
    <div className="dogList">
      <h2> Dog List </h2>
      <Dog />
      <Dog />
      <Dog />
    </div>
  );
```